ProjectSettings/TagManager.asset

```diff
@@ -10,6 +10,9 @@ TagManager:
   - SpawnPlatform
   - GoalPlatform
   - NoclipEnabler
+  - SpawnPoint
```

Comment:
To me, it does not seem that this tag is being used (same goes for the `SpawnPoint` element) in the scene. I would remove this if that is the case. If it is used, I apologise :)